lib/modules/schedule/page/schedule_home_page.dart

```diff
@@ -0,0 +1,164 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:ui_collection/modules/schedule/domain/bloc/schedule_bloc.dart';
+import 'package:ui_collection/modules/schedule/widget/drawer/app_drawer_widget.dart';
+import 'package:ui_collection/modules/schedule/widget/home_widgets/menu_card_widget.dart';
+
+import '../../../widget/colors/common_color.dart';
+import '../domain/entities/home_card_entity.dart';
+import '../domain/entities/schedule_item_entity.dart';
+import '../widget/home_widgets/clocking_widget.dart';
+import '../widget/home_widgets/header_widget.dart';
+
+class ScheduleHomePage extends StatefulWidget {
+  const ScheduleHomePage({Key? key}) : super(key: key);
+
+  @override
+  State<ScheduleHomePage> createState() => _SchedulePageState();
+}
+
+class _SchedulePageState extends State<ScheduleHomePage> {
+  ScheduleItemEntity getToday(List<ScheduleItemEntity> schedules) =>
```

Comment:
Method Rule #3, Always place similar or related functions next to each other.

Move this method after the build method.